chronograf.go

```diff
@@ -702,6 +704,56 @@ type LayoutsStore interface {
 	Update(context.Context, Layout) error
 }
 
+// ProtoboardMeta is the metadata of a Protoboard
+type ProtoboardMeta struct {
+	Name             string `json:"name"`
+	Icon             string `json:"icon,omitempty"`
+	Version          string `json:"version"`
+	DashboardVersion string `json:"dashboardVersion"`
+	Description      string `json:"description,omitempty"`
+	Author           string `json:"author,omitempty"`
+	License          string `json:"license,omitempty"`
+	URL              string `json:"url,omitempty"`
+}
+
+// ProtoboardCell holds visual and query information for a cell
+type ProtoboardCell struct {
```

Comment:
Why cant we reuse the dashboard cell data structure here?

Reply:
Protoboard doesn't have `id` -- thoughts?

Reply:
just `json:"id,omitempty"` on the `DashboardCell`

Reply:
we'll add this and add validation code on dashboards. the aim was to prevent dashboardcells from ever getting into a state where they didn't have an id